simple_app/utils.py

```diff
@@ -491,32 +491,32 @@ def absmags(df: pd.DataFrame, all_bands: np.ndarray) -> pd.DataFrame:
         The output dataframe with absolute mags calculated
     """
 
-    def pogsonlaw(m: Union[float, np.ndarray], dist: Union[float, np.ndarray]) -> Union[float, np.ndarray]:
+    def pogsonlaw(m: Union[float, np.ndarray]) -> Union[float, np.ndarray]:
         """
         Distance modulus equation
```

Comment:
Could you comment this more. It's more than just the distance modulus equation. It uses the parallax to calculate a distance. But returns a Nan if the distance is 0 or negative?